MAUI/Badge-View/Badge-customization.md

```diff
@@ -344,3 +344,46 @@ Content = sfBadgeView;
 {% endtabs %}
 
 ![BadgeView Alignment](badge-customization_images/badge_alignment.png)
+
+## Visibility of badge
+
+The IsVisible property is used to show or hide the badge in the BadgeView. By default, visibility of the badge is enabled.
+
+N> The badge appears only when the `IsVisible` property value is set to **true**.
```

Comment:
@KarthikaRavichandranSF4617, its not needed, you can remove it.

Reply:
I have removed it.